src/status_im/communities/core.cljs

```diff
@@ -793,4 +793,4 @@
                                {:error        %
                                 :community-id community-id
                                 :public-key   public-key
-                                :role-id      role-id})]})
+                                :role-id      role-id})]})
```

Comment:
It'd be better if you could keep the trailing newline character, since we use the Clojure Style Guide.